addons/hr_holidays/models/hr_leave.py

```diff
@@ -1543,13 +1547,13 @@ def activity_update(self):
             if holiday.state == 'draft':
                 to_clean |= holiday
             elif holiday.state == 'confirm':
-                holiday.activity_schedule(
+                holiday.with_context(short_name=False).activity_schedule(
```

Comment:
This I don't get :eyes: Maybe explain what is the issue?

Reply:
The `short_name` context is within the menu of the overview ([calendar view](https://github.com/odoo/odoo/blob/16.0/addons/hr_holidays/views/hr_leave_views.xml#L610)) and when you create a new time off and therefore send it, only the `short_name` of `name_get` is delivered which is skipping the important date. So, this little magic context makes the behavior of the notification inconsistent.
As it is the only way how to control the message of the `mixin`, we make sure here that this context key is not applied.

Reply:
agreed, well spotted

Reply:
2/3 ✅ 🥳